assets/components/src/plugin-installer/index.js

```diff
@@ -21,6 +21,11 @@ const PLUGIN_STATE_ACTIVE = 1;
 const PLUGIN_STATE_INSTALLING = 2;
 const PLUGIN_STATE_ERROR = 3;
 
+/**
```

Comment:
That's a great idea. I'm not quite sure how to do this though since the className doesn't get changed:

![Screenshot 2019-12-12 at 09 36 46](https://user-images.githubusercontent.com/177929/70700721-12eddb80-1cc3-11ea-9e8a-f94be6875b14.png)

Reply:
Just pushed a change to add a class based on installation status: https://github.com/Automattic/newspack-plugin/pull/322/commits/23f903f4441ca4cb4e9e740444178fcc3aee96d1. The options are:

- newspack_plugin-installer__status_none
- newspack_plugin-installer__status_active
- newspack_plugin-installer__status_installing
- newspack_plugin-installer__status_error

(And of course feel free to change the class names if they aren't compliant with best practices)

Reply:
Oh brilliant. Thank you. I will take care of the error now 😊

Reply:
Fixed in 89e9b97